src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -106,15 +119,21 @@ public ObservableList<Person> getPersonList() {
         return persons.asUnmodifiableObservableList();
     }
 
+    @Override
+    public Person getUser() {
+        return user;
+    }
+
     @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof AddressBook // instanceof handles nulls
-                && persons.equals(((AddressBook) other).persons));
+                && persons.equals(((AddressBook) other).persons)
+                && user.equals(((AddressBook) other).user));
     }
 
     @Override
     public int hashCode() {
-        return persons.hashCode();
+        return persons.hashCode() + user.hashCode();
```

Comment:
I would like to seek clarification for this matter: Is there a possibility of hash collision here?

Reply:
I'm not sure how else to do that. Lemme check.